tests/OptionsTest.php

```diff
@@ -243,4 +244,24 @@ public function excludedPathProviders()
             [__FILE__, __FILE__],
         ];
     }
+
+    /**
+     * @dataProvider maxBreadCrumbsInvalidValuesDataProvider
+     */
+    public function testMaxBreadCrumbsInvalidValues($invalidValue): void
```

Comment:
This test is only testing the invalid values but we should at least test the min/max allowed values which are `0` and `Options::DEFAULT_MAX_BREADCRUMBS`. I would then rename this method something like `testMaxBreadcrumbsOptionIsValidatedCorrectly` and its data provider `maxBreadcrumbsOptionIsValidatedCorrectlyDataProvider`. The `expectException` should be then wrapped in an `if` statement according to the validity of the input value

Reply:
> it helped me spotting that the test was also invalid because it was wrapping the value inside an array

Nice I didn't even notice that 😄